src/init.cpp

```diff
@@ -932,10 +932,6 @@ bool AppInitParameterInteraction(const ArgsManager& args, bool use_syscall_sandb
     }
 
     // mempool limits
-    int64_t nMempoolSizeMax = args.GetIntArg("-maxmempool", DEFAULT_MAX_MEMPOOL_SIZE_MB) * 1000000;
-    int64_t nMempoolSizeMin = args.GetIntArg("-limitdescendantsize", DEFAULT_DESCENDANT_SIZE_LIMIT_KB) * 1000 * 40;
```

Comment:
In efcbd12767d87b33b6c6deb86d441292ff681b13 "init: Remove redundant -*mempool*, -limit* queries "

Seems like you've dropped a `* 40` here? Is that maybe what's causing the CI failure?